config.json.dist

```diff
@@ -7,8 +7,8 @@
 	"__comment1": "IP players should connect to to join your servers, should be portforwarded and stuff",
 	"publicIP": "localhost",
 	
-	"__comment2": "IP the master web server runs at, can be ignored on the master system",
-	"masterIP": "localhost",
+	"__comment2": "URL the master web server runs at, can be ignored on the master system.  Must not end with a slash",
+	"masterURL": "http://localhost:8080",
```

Comment:
Hm, maybe we should not add port here, cause we have the "masterPort" setting and you need to change both - that can be error-prone and a bit annoying.
Maybe in the code, we should just combine config.masterURL = config.masterURL + config.masterPort.toString()` or use variables like `masterHost` and `masterPort` and combine them into `masterURL` ?

Reply:
masterPort is only for the master server and specifies what port to listen on, masterURL is only for slaves and specifies where to connect.  It should be really clear after the config a split into two.

I don't think we should do any combining of host and port into the url because that limits your option on how to host the master.  With a url you can specify for example `https://example.com/master` (notice how port isn't needed when running on the default one) and proxy that to the master server with for example apache or nginx as the front-end.  This could also be used to serve all the files that doesn't need any template logic once those have been separated out.